client/styles/components/_sidebar.scss

```diff
@@ -58,7 +58,7 @@
   @include themify() {
     color: map-get($theme-map, 'inactive-text-color');
     &:hover > .file-item__content .sidebar__file-item-name {
-      color: map-get($theme-map, 'primary-text-color');
+      color: $p5js-pink;
```

Comment:
could you pull this color from the theme map so it looks good in the light dark, and high contrast themes?